firebase-case-study.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Firebase Case Study
+keywords: development platforms, firebase, software development kit
+tags: Development Platforms, firebase, software development kit
+sidebar: development-platforms
+permalink: development-platforms/example-product.html
+folder: development-platforms
+---
+
+## Introduction
+What is Firebase? Main purpose
+
+Firebase is a Mobile Application Development Platform with tools to make the job a developer would otherwise have to build themselves easier. This includes everything from analytics, authentication, databases, configuration, file storage, push messaging and more! The service is hosted on a cloud solution provided by Google, and there are backend components completely maintained and operated by them. There is no need to use any middleware between your app and the service, as the client Software Development Kits provided by firebase use the backend services directly. Some might call Firebase a “back-end as a service”, but Doug Stevenson, who works with Firebase, has not felt comfortable wedging it fully into that category.
+
+## Brief History
+History of firebase (2011 - today)
+
+In 2011 James Tamplin and Andrew Lee founded Envolve. Envolve allowed for integration of chat functionality via provided API for developers to use into their websites. After release they found that it was being used to pass application data, and not just text messages. Tamplin and Lee then founded Firebase as a separate company in September 2011. Their first product was Firebase Real-Time Database and is an API that synchronizes data across Android, iOS and Web Applications with Firebase’s cloud.  After launching the beta they raised 1.1M USD in seed funding in May 2012, and a year later, in 2013 they raised five times as much in Series A funding.
+
+
+In 2014 they launched Firebase Hosting and Firebase Authentication as two new products. Later that year, in October, Google purchased Firebase and started further development and maintenance of the Brand. Google then acquired Divshot (2015), an interface builder and web hosting platform to merge into the Firebase Team. In the following years they expanded the service further and integrated AdMob, Google Platform and Google Ads (2016), as well as acquired Fabric and Crashlytics from Twitter (2017). They’ve expanded their service through the years and there are now several options to build, improve and grow an application with Firebase. 
+
+![image](https://user-images.githubusercontent.com/91533917/218750092-e8d847d7-538b-414e-9d04-83966b1a886b.png)
+
+
+## Features
```

Comment:
Suggestion:
I think it will be useful to add some explanation to each feature, one or two sentences. Or maybe explain in greater detail some of the features you find most helpful as a front-end developer.